pallets/phala/src/compute/base_pool.rs

```diff
@@ -0,0 +1,1135 @@
+pub use self::pallet::*;
+
+use sp_runtime::traits::{AtLeast32BitUnsigned, Zero};
+
+#[frame_support::pallet]
+pub mod pallet {
+	#[cfg(not(feature = "std"))]
+	use alloc::format;
+	#[cfg(feature = "std")]
+	use std::format;
+	use crate::computation;
+	use crate::wrapped_balances;
+	use crate::pool_proxy::*;
+	use crate::registry;
+	use crate::vault;
+	use crate::BalanceOf;
+
+	pub use rmrk_traits::{
+		primitives::{CollectionId, NftId},
+		Nft, Property,
+	};
+
+	use super::{extract_dust, is_nondust_balance};
+
+	use frame_support::{
+		pallet_prelude::*,
+		traits::{
+			tokens::fungibles::Transfer, tokens::nonfungibles::InspectEnumerable, StorageVersion,
+			UnixTime,
+		},
+	};
+
+	use fixed::types::U64F64 as FixedPoint;
+	use fixed_macro::types::U64F64 as fp;
+
+	use crate::balance_convert::{div as bdiv, mul as bmul, FixedPointConvert};
```

Comment:
Please remove the unnecessary space lines

Reply:
done